mlir/test/Dialect/ArmSME/arm-sme-to-llvm.mlir

```diff
@@ -0,0 +1,401 @@
+// RUN: mlir-opt %s -convert-vector-to-llvm="enable-arm-sme" -cse -canonicalize -split-input-file -verify-diagnostics | FileCheck %s
+
+// Test conversion of higher-level ArmSME ops to LLVM intrinsics.
```

Comment:
By "higher-level" you basically mean the custom ops (as opposed the intrinsics), right? As in, there are 2 categories ATM, correct?

Reply:
Not really sure how I can make this any clearer than it is

Reply:
Perhaps just drop "higher-level"? That's the bit that confused me.

Reply:
Done